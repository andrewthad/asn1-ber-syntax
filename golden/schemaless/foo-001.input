300E0203 0186A006 072B0601 02010101
