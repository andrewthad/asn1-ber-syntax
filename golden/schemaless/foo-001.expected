Value
    { tagClass = Universal
    , tagNumber = 16
    , contents = Constructed
        ( fromListN 2
            [ Value
                { tagClass = Universal
                , tagNumber = 2
                , contents = Primitive
                    [ 0x01
                    , 0x86
                    , 0xa0
                    ]
                }
            , Value
                { tagClass = Universal
                , tagNumber = 6
                , contents = Primitive
                    [ 0x2b
                    , 0x06
                    , 0x01
                    , 0x02
                    , 0x01
                    , 0x01
                    , 0x01
                    ]
                }
            ]
        )
    }
